qrenderdoc/CMakeLists.txt

```diff
@@ -227,6 +227,12 @@ if(CMAKE_CXX_COMPILER_ID MATCHES "Clang" AND CMAKE_CXX_COMPILER_VERSION VERSION_
         "QMAKE_CXXFLAGS+=-Wno-comma\n")
 endif()
 
+if(CMAKE_CXX_COMPILER_ID MATCHES "GNU" AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 14.0)
```

Comment:
Any particular reason this change is not part of the existing `if(CMAKE_COMPILER_IS_GNUCXX)` clause with a version check i.e.

```
if(CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 14.0)
...
...
endif()
```